scripts/grest-helper-scripts/setup-grest.sh

```diff
@@ -211,6 +211,23 @@ else
   err_exit "ERROR!! ${CNODE_HOME}/files/config.json not found! Please ensure you've run pre-requisites!"
 fi
 
+read -ra SHGENESIS <<< $(jq -r '[
+  .activeSlotsCoeff,
+  .updateQuorum,
+  .networkId,
+  .maxLovelaceSupply,
+  .networkMagic,
+  .epochLength,
+  .systemStart,
+  .slotsPerKESPeriod,
+  .slotLength,
+  .maxKESEvolutions,
+  .securityParam
+  ] | @tsv' < "${SHGENFILE}")
+
+#PS: Given the Plutus schema is far from finalized, we expect changes as SC layer matures and PAB gets into real networks. For now, dumping compressed jq (will be inserted as shell escaped json data blob, can be changed in future iterations - many of these will already are part of pParams and may be eliminated).
```

Comment:
Split this comment into multiple lines for readability?

Reply:
Done